src/cls/_ZPM/PackageManager.cls

```diff
@@ -1014,16 +1071,16 @@ ClassMethod Init(ByRef pCommandInfo) [ Internal ]
 	
 	If (tOldSourceControlClass '= "") && (tOldSourceControlClass '= tCompositeSCClass) && (tOldSourceControlClass '= tPackageMgrSCClass) {
 		Write !,"Enabling package manager extension... "
-		$$$ThrowOnError(##class(%ZPM.PackageManager.Developer.Extension.Composite).SetExtensionClasses(tOldSourceControlClass,$ListBuild(tPackageMgrSCClass)))
+		$$$ThrowOnError($Classmethod(tCompositeSCClass, SetExtensionClasses(tOldSourceControlClass,$ListBuild(tPackageMgrSCClass))))
```

Comment:
This call is incorrect. Need to pass separate arguments when using $ClassMethod:
```
$ClassMethod(tCompositeSCClass, "SetExtensionClasses", tOldSourceControlClass, $ListBuild(tPackageMgrSCClass))
```